File: C/maths.c
#include <stdio.h>

#include "Includes/mathsV.h"

int main(void){

    int num1,num2,val;

    printf ("Enter 2 numbers to be added : ");
    
    scanf("%d%d", &num1,&num2);
    
    printf("\nThe numbers you want added is %d and %d\n", num1,num2);
    
    val = add(num1 , num2);

    printf("\n%d + %d = %d\n", num1 , num2 , val);

    val = sub(num1 , num2);

    printf("\n%d - %d = %d\n", num1 , num2 , val);
}

File: Javascript/maths.js
function add(a,b){
 
    return a + b;
}

"use strict";

const ps = require("prompt-sync");
const prompt = ps();

console.log("Please enter two numbers to add");

var num1 = parseInt(prompt("Enter number 1 to add : "));
var num2 = parseInt(prompt("Enter the second number to add : "));

var val = add(num1, num2);


console.log(`${num1} + ${num2} = ${val}`);